src/Agent.Plugins/TestResultParser/TraceLogger.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Agent.Plugins.Log.TestResultParser.Contracts;
+using Agent.Sdk;
+
+namespace Agent.Plugins.TestResultParser.Plugin
+{
+    public class TraceLogger : ITraceLogger
+    {
+        public TraceLogger(IAgentLogPluginContext context)
+        {
+            _context = context;
+        }
+
+        #region interface implementation
+
+        /// <inheritdoc />
+        void ITraceLogger.Warning(string text)
+        {
+            _context.Output($"##vso[task.logissue type=warning;]{text}");
+        }
+
+        /// <inheritdoc />
+        void ITraceLogger.Error(string error)
+        {
+            _context.Output($"##vso[task.logissue type=error;]{error}");
```

Comment:
do these ##vso make sense, i didn't see the finalize logs changing color based on these, i don't think the these logs get parsed for commands/log level

Reply:
while it doesn't colorize currently, its purely how agent implementation detail. 
As a plugin, we should write in the documented format